test/topology_random_failures/error_injections.py

```diff
@@ -27,3 +27,11 @@
     "stop_after_streaming",
     "stop_after_bootstrapping_initial_raft_configuration",
 )
+
+# Error injections which can cause a node's hang due to some timeouts.
```

Comment:
I expect the annotation here to also reference issues as well. Anything that is causing a node to hang is most likely a bug, right?

Reply:
It's an expected behavior, actually, not a bug. What we do in the test is ensure that this hang does not affect the cluster.

Here little bit more details: https://github.com/scylladb/scylladb/issues/18638

Reply:
I don't understand. It's still a test bug to "hang" a node and then not handle it.
In the past, I had this problem in one of the "topology" framework tests I wrote - and I was missing a proper API to kill an "hung" node without resulting in later errors. I think these missing features in the test framework, or bugs in tests, should be fixed by giving tests the ability to do what they need - not by creating blacklists of tests in the test framework.

Reply:
The test does `server_stop()` on the hung node.